### ISSUE TITLE
GRE-ME conversion fails: "cannot find sSUM scan" (Prisma dataset)

### ISSUE THREAD

**Issue by DPappFMRIB** [neuropoly/bids_neuropoly#5]
Acq details: Prisma scanner, VE11C software, protocol imported via EXAR file (recent as of Dec 2018), no problems during acquisition. Conversion via DCM2NIIX2, or SPM/DICOM convert runs without error.

However, conversion using the provided code fails with the following warning.
"WARNING: Detected multiple GRE-ME scans. Only keeping the file which contains "sum".
WARNING: Cannot find sSUM scan."

Yet, only one GRE-ME dataset is present. 

Full terminal dump:
https://pastebin.com/1Vzu9wrS

**Comment by jcohenadad** [comment]
@DPappFMRIB thank you for the feedback. Are you using the latest version (i.e. commit 52c3d6798c593a4c9610a2886a681992a772dc01)?

If so, do you mind sending me the Dicom folder so I can try to reproduce the issue on my end?

**Comment by DPappFMRIB** [comment]
@jcohenadad 

I am using the most recent version, yes, and I have sent you the folder.

**Comment by jcohenadad** [comment]
@DPappFMRIB I tried with your data and cannot reproduce the bug. Could you please copy/paste the content of the script so i can double check you are running the latest version of the script?

**Comment by DPappFMRIB** [comment]
@jcohenadad Pasted in the link below. As far as I can tell, it is identical to your latest commit (52c3d67), as per text comparison. Sorry for taking up your time.

https://pastebin.com/9vuDGS9Y

**Comment by jcohenadad** [comment]
@alexfoias can you please try to reproduce the issue using the exact same version of dcm2niix (MacOS)?
~~~
Chris Rorden's dcm2niiX version v1.0.20171215 (OpenJPEG build) Clang8.1.0 (64-bit MacOS)
~~~

**Comment by alexfoias** [comment]
@jcohenadad I used the same dcm2niiX version as @DPappFMRIB  but I couldn't reproduce the bug.

Here is my terminal output: https://pastebin.com/m4Gmv56P

**Comment by jcohenadad** [comment]
in that case let's fix https://github.com/neuropoly/bids_neuropoly/issues/4, and then we can properly address this issue.

**Comment by jcohenadad** [comment]
@DPappFMRIB could you please try doing the following:
- go to the `bids_neuropoly` repository then run:
~~~
git pull
pip install -e .
~~~
- then re-run `convert_dcm2nii.py` in the same dataset that produced the issue described in this thread.
- in the output, a log file called `bids_neuropoly_logger.log` should be created. Please dump it in pastebin (or equivalent).
thanks!

**Comment by DPappFMRIB** [comment]
@jcohenadad 

Sadly, the same issue arises. I had a colleague of mine try and run the conversion on the same data, it worked for them. I suspect the issue lies with my system. Thank you for your help!

Terminal and logger output:
https://paste.ee/p/61zxL#UiZBhvBznlsKmB9TUUnQdFEkuzbHsnSU

**Comment by jcohenadad** [comment]
@DPappFMRIB thanks for the feedback. I've added more verbose to help us figure out what's happening on your system. Could you please try again with the latest version (24374d9ec8165d3260d1f5722290b0af8e8ca15d). Thanks!

**Comment by DPappFMRIB** [comment]
@jcohenadad 

Sad to say the issue still applies.

Log here:
https://paste.ee/p/GAvAk#SSsySPHq2lTDgbnxJOfAyD1aiiVRfHEm

**Comment by jcohenadad** [comment]
@DPappFMRIB yes this was expected as I didn't touch the code-- i only added verbose to understand what was happening on your end. And we made progress :-)

Interestingly, `dcm2niix` forces output file names to be `datax.*`:
~~~
INFO:root:List of converted files:
/var/folders/ff/px4nh9y56fb6kxfmwdks37bc0000gr/T/tmpfVF2Vg/datab.nii.gz
/var/folders/ff/px4nh9y56fb6kxfmwdks37bc0000gr/T/tmpfVF2Vg/data.nii.gz
/var/folders/ff/px4nh9y56fb6kxfmwdks37bc0000gr/T/tmpfVF2Vg/datah.nii.gz
/var/folders/ff/px4nh9y56fb6kxfmwdks37bc0000gr/T/tmpfVF2Vg/datad.nii.gz
/var/folders/ff/px4nh9y56fb6kxfmwdks37bc0000gr/T/tmpfVF2Vg/dataf.nii.gz
/var/folders/ff/px4nh9y56fb6kxfmwdks37bc0000gr/T/tmpfVF2Vg/datac.nii.gz
/var/folders/ff/px4nh9y56fb6kxfmwdks37bc0000gr/T/tmpfVF2Vg/dataa.nii.gz
/var/folders/ff/px4nh9y56fb6kxfmwdks37bc0000gr/T/tmpfVF2Vg/datae.nii.gz
/var/folders/ff/px4nh9y56fb6kxfmwdks37bc0000gr/T/tmpfVF2Vg/datag.nii.gz
~~~

Whereas on our end it looks more like this:
~~~
INFO:root:List of converted files:
/var/folders/6f/wy6ljmx9453cgth2qwv5l1s80000gn/T/tmpWLNCUf/001_GRE-MT1_20190116164758_16.nii.gz
/var/folders/6f/wy6ljmx9453cgth2qwv5l1s80000gn/T/tmpWLNCUf/001_GRE-T1w_20190116164758_18.nii.gz
/var/folders/6f/wy6ljmx9453cgth2qwv5l1s80000gn/T/tmpWLNCUf/001_T2w_20190116164758_13.nii.gz
/var/folders/6f/wy6ljmx9453cgth2qwv5l1s80000gn/T/tmpWLNCUf/001_DWI_b0_20190116164758_14.nii.gz
/var/folders/6f/wy6ljmx9453cgth2qwv5l1s80000gn/T/tmpWLNCUf/001_T1w_20190116164758_12.nii.gz
/var/folders/6f/wy6ljmx9453cgth2qwv5l1s80000gn/T/tmpWLNCUf/001_GRE-ME_20190116164758_19.nii.gz
/var/folders/6f/wy6ljmx9453cgth2qwv5l1s80000gn/T/tmpWLNCUf/001_GRE-MT0_20190116164758_17.nii.gz
~~~

Maybe it is related to your OS version (18.2.0). 
@alexfoias Can you try to reproduce with that OS version?

**Comment by alexfoias** [comment]
@jcohenadad I tried on Darwin 18.0. It worked, I had no issue. I'm doing the system update to get to 18.2.0.

Here is my output: https://paste.ee/p/ZDshZ

**Comment by jcohenadad** [comment]
@alexfoias any update?

**Comment by alexfoias** [comment]
I tried it Darwin 18.2.0 and everything seems to work well.

Here is my output: https://pastebin.com/GC7JxFcu

**Comment by jcohenadad** [comment]
OK, I'm out of ideas. Sorry @DPappFMRIB but we are not able to reproduce this error, making it difficult to find a fix for it.
